bin/openstudio_meta

```diff
@@ -326,10 +362,16 @@ class StartLocal
   # @return [Fixnum] Return status
   #
   def execute(sub_argv)
-    options = { debug: false, force_directories: false, workers: 1 }
+    mongod = which 'mongod'
```

Comment:
eh. it is just a convenience. if mongo is in the path and a mongo option was not passed, then just use what you find in the path.

PAT will obsoletely pass an explicit path to mongo. I just got tired of typing out all of theses paths in testing.